### PR TITLE
Add ipmitool to SONIC rootfs

### PR DESCRIPTION
**Pull request by jostar-yang**
**- What I did**
add ipmitool to rootfs
**- How I did it**
add ipmitool to build_debian.sh
**- How to verify it**
User can use ipmitool in rootfs
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: build_debian.sh [hunk 1]
```diff
@@ -221,7 +221,8 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     unzip                   \
     gdisk                   \
     sysfsutils              \
-    grub2-common
+    grub2-common            \
+    ipmitool
```
Comment (lguohan) [inline build_debian.sh]: can we move this into pmon docker? will it work?

Comment (lguohan) [reply build_debian.sh]: @jleveque

Comment (jostar-yang) [reply build_debian.sh]: Ipmitool is common utility.  It should put to "/usr/bin" for user application can access BMC.
If there is any concern on SONIC project,  please tell me how to add ipmitool cmd to pmon docker.  
I need to study and test on as7716-32XB  BMC project. Thanks.

Comment (jostar-yang) [reply build_debian.sh]: We study pmon docker and ipmitool use.  We think ipmitool is like i2c-tool that are common utility tool.  Ipmitool should put to /usr/bin or usr/sbin in SONiC rootfs, not put to any docker container. For BMC project, user only can access i2c from ipmitool. For non-MBC project, user can access i2c from i2c-tool.

**Comment by lguohan** [comment]
ipmitool has been added to docker-platform-monitor.

**Pull request closed without merging**